### PR TITLE
fan page part 1, Top Gun

### PR DESCRIPTION
**Pull request by lanceaune**
Completed fan page 1. Top Gun

### CODE REVIEW

**Comment by lanceaune** [comment]
updated file to include footer

File: Topgun2.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (mbMosman) [inline Topgun2.html]: This needs to be called index.html - It is the page used by default for the main page of a website by most web servers.

**Comment by mbMosman** [comment]
A few things to fix up, but a good start on the page.  

You've got validation errors which would be a large point reduction on a project, so make sure to double check your validation once you get your site live on GitHub.

image

File: Topgun2.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+<title>Top Gun</title>
+<meta charset="utf-8">
```
Comment (mbMosman) [inline Topgun2.html]: Head with title and charset? :heavy_check_mark:

File: Topgun2.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+</head>
+<body>
+    <h1>TOP GUN</h1>
+    <hr>
+    <h2>The movie Top Gun</h2>
+    <p>Ever since i was a kid, i loved watching this movie. Everything from the awesome stunts the fighter pilots were doing to being a bad ass who helps keep America safe. Top Gun was simply an awesome movie. The three things i loved about this move were: <ol> <li>Fighter jets</li><li> Flying</li><li>Aircraft Carriers</li></ol></p>
```
Comment (mbMosman) [inline Topgun2.html]: You can't put a list inside a paragraph, you need to end the paragraph and then start the list.

A list of something?  :heavy_check_mark:

File: Topgun2.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <hr>
+    <a href= "https://en.wikipedia.org/wiki/Top_Gun"> Top Gun Wiki</a>
+    <footer> &copy;Copyright 2016 LA Company. All Rights Reserved
+                    <a href="mailto:la@lacompany.com">la@lacompany.com</a></footer>
```
Comment (mbMosman) [inline Topgun2.html]: Footer w/ copyright and email?  :heavy_check_mark:

File: Topgun2.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Top Gun</title>
+<meta charset="utf-8">
+</head>
+<body>
+    <h1>TOP GUN</h1>
+    <hr>
+    <h2>The movie Top Gun</h2>
```
Comment (mbMosman) [inline Topgun2.html]: Should have at least 2 subheadings....

File: Topgun2.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Top Gun</title>
+<meta charset="utf-8">
+</head>
+<body>
+    <h1>TOP GUN</h1>
```
Comment (mbMosman) [inline Topgun2.html]: Should have a `<header>` around the `<h1>`

**Pull request closed without merging**